Tests.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <assert.h>
+#include "ColorPair.h"
+#include "PrintColorPair.h"
+
+
+namespace TelCoColorCoder
+{
+    void testNumberToPair(int pairNumber,
+        TelCoColorCoder::MajorColor expectedMajor,
+        TelCoColorCoder::MinorColor expectedMinor)
+    {
+        TelCoColorCoder::ColorPair colorPair =
+            GetColorFromPairNumber(pairNumber);
+        std::cout << "Got pair " << colorPair.ToString() << std::endl;
+        assert(colorPair.getMajor() == expectedMajor);
+        assert(colorPair.getMinor() == expectedMinor);
+    }
+
+    void testPairToNumber(
+        TelCoColorCoder::MajorColor major,
+        TelCoColorCoder::MinorColor minor,
+        int expectedPairNumber)
+    {
+        int pairNumber = GetPairNumberFromColor(major, minor);
+        std::cout << "Got pair number " << pairNumber << std::endl;
+        assert(pairNumber == expectedPairNumber);
+    }
+    void testPrint()
+    {
+        PrintColorPair s;
+        s.printManual();
```

Comment:
This test has no assert, so the entire effort to test it becomes manual - checking if there are 25 rows, ensuring none of them are duplicated, etc.
It's not necessary to eliminate manual testing, but is there any way we can reduce the number of things to check manually? Give it a thought. More about this in the next assignment

Reply:
Added a method that'll return the string that'll be printed, added test for the same